mirdata/gtzan_genre.py

```diff
@@ -73,12 +86,48 @@ def audio(self):
         return load_audio(self.audio_path, sample_rate=22050)
 
     def to_jams(self):
-        """(Not Implemented) Jams: the track's data in jams format"""
-        raise NotImplementedError
+        """Jams: the track's data in jams format"""
+        # Initialize top-level JAMS container
+        jam = jams.JAMS()
```

Comment:
This should use the [`mirdata.jams_utils.jams_converter`](https://github.com/mir-dataset-loaders/mirdata/blob/master/mirdata/jams_utils.py#L12) function. If it's missing fields, we should add them to the converter.

Reply:
@rabitt so i should make a `gtzan_to_jams` function in `jams_utils`? and pass a `gtzan_data` (default `None`) to the `jams_converter`? that sounds overkill to me, but i can live with that.